consul/src/main/scala/io/buoyant/consul/v1/KvApi.scala

```diff
@@ -9,13 +9,13 @@ import com.twitter.finagle.stats.{DefaultStatsReceiver, StatsReceiver}
 import com.twitter.util._
 
 object KvApi {
-  def apply(c: Client): KvApi = new KvApi(c, s"/$versionString")
+  def apply(c: Client, backoff: Stream[Duration]): KvApi = new KvApi(c, s"/$versionString", backoff)
```

Comment:
Should the default value removed from the constructor be added here? (i.e, 
```scala
def apply(
  c: Client,
  backoff: Stream[Duration] = Backoff.exponentialJittered(1.milliseconds, 5.seconds)
): KvApi = new KvApi(c, s"/$versionString", backoff)
```

Reply:
this would also make it no longer necessary to change all the tests in `KVApiTest`...

Reply:
I like that the default is defined in one place only.

Reply:
Yeah, I saw that after scrolling through the rest of the PR; disregard that.